### PR TITLE
feat(config): update Zooz ZEN20 with additional parameters 28 - 36

### LINKED ISSUE #4806: Missing Configuration Parameters for Zooz Zen20
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

I received the below configuration parameters from Zooz tech support after inquiring about what I initially believed to be missing functionality. As it turns out, the ability to track Amps and Voltage via the Zen20 is turned off by default. In order to enable this functionality, the parameters listed below need to be configured.

In my case, I leveraged the Custom Configuration feature of ZwaveJS2Mqtt to test and enact these configuration changes. Figured I'd share the information Zooz provided me so that it can be included in the main config for this device.

> Electrical Current (A) Report Frequency
> Parameter 35: The number entered as value corresponds to the number of seconds. So if 30 is entered, the Strip will report electrical current values (Amperes) every 30 seconds.
> 
> Values: 0 – electrical current reports disabled (default); 30 – 2678400.
> 
> Size: 4 byte dec
> 
> Voltage Report (V) Frequency
> Parameter 36: The number entered as value corresponds to the number of seconds. So if 30 is entered, the Strip will report Voltage values (Volts) every 30 seconds.
> 
> Values: 0 – voltage reports disabled (default); 30 – 2678400.
> 
> Size: 4 byte dec

### Device information

Manufacturer: Zooz
Model name: Zen20

Github Page: https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x015d/zen20.json

### How are you using `node-zwave-js`?

- [ ] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version:
`node-zwave-js` branch:
`zwavejs2mqtt` branch:

### Did you change anything?

no

### If yes, what did you change?

_No response_

### Did this work before?

Don't know, this is a new device

### If yes, where did it work?

_No response_

### Attach Driver Logfile

2022-07-14 14:39:57.076 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 32.6 => 34.9
2022-07-14 14:39:57.173 INFO ZWAVE: Node 65: value updated: 50-0-value-66049 32.6 => 34.9
2022-07-14 14:39:57.765 INFO ZWAVE: Calling api sendCommand with args: [
{ nodeId: 65, commandClass: 112 },
'set',
[ 35, 30, 4, [length]: 3 ],
[length]: 3
]
2022-07-14 14:40:03.077 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 34.9 => 27.3
2022-07-14 14:40:03.106 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.170 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.272 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.338 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.385 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.479 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.672 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.787 INFO ZWAVE: Node 36: value updated: 50-0-value-66049 0 => 0
2022-07-14 14:40:04.901 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.961 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:05.030 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3

### PR DESCRIPTION
**Pull request by alvinchen1**
Per https://www.support.getzooz.com/kb/article/1024-zen20-power-strip-advanced-settings/

Resolve https://github.com/zwave-js/node-zwave-js/issues/4806 and believe with commit, can close

I looked at https://github.com/zwave-js/node-zwave-js/issues/4808 but was not sure if this helps or not.  I do have one of the new 3.0 devices so can provide some investigation around it, but do not have a ver 2.0 to supply firmware or differentiation.

### CODE REVIEW

File: packages/config/config/devices/0x027a/zen20.json [hunk 1]
```diff
@@ -162,6 +162,143 @@
...
+			"options": [
+				{
+					"label": "Disable all power monitoring and reporting",
+					"value": 0
+				},
+				{
+					"label": "Enable all power monitoring and reporting",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen20.json]: According to https://www.support.getzooz.com/kb/article/303-zen20-power-strip-change-log/, those were added in firmware 2.2.

Also we should simplify the wording and make use of the enable/disable template
```suggestion
			"$if": "firmwareVersion >= 2.2",
			"$import": "~/templates/master_template.json#base_enable_disable",
			"label": "Power/Energy Monitoring",
			"defaultValue": 1
```

Make it similar for the individual channels, just with a different label: "Power/Energy Monitoring: Relay 1", "...Relay 2" and so on.

File: packages/config/config/devices/0x027a/zen20.json [hunk 2]
```diff
@@ -162,6 +162,143 @@
...
+				},
+				{
+					"label": "Enable USB Ports power monitoring and reporting",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "35",
+			"label": "Current Report (A) Frequency",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen20.json]: According to https://www.support.getzooz.com/kb/article/303-zen20-power-strip-change-log/ this and the next one were added in 2.3
```suggestion
			"$if": "firmwareVersion >= 2.3",
			"label": "Current (A) Report Frequency",
```
Also, having (A) before Report is consistent with the first parameters.

File: packages/config/config/devices/0x027a/zen20.json [hunk 3]
```diff
@@ -162,6 +162,143 @@
...
+			"unit": "seconds",
+			"valueSize": 4,
+			"minValue": 5,
+			"maxValue": 2678400,
+			"defaultValue": 30
+		},
+		{
+			"#": "36",
+			"label": "Voltage Report (V) Frequency",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen20.json]: Dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, this needs a couple of changes though.

**Review by AlCalzone — APPROVED** [review]
I did a few minor changes for consistency.

**Comment by alvinchen1** [comment]
> I did a few minor changes for consistency.

Ah, I see, thanks! I also am a stickler for consistent style as well, so appreciate those changes.  Also appreciate that you grouped the (A) and (V) together with the (W) and (kWH).  Makes sense.
